Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM praekeltfoundation/django-bootstrap:py3.6-jessie
-RUN apt-get-install.sh git \
-    redis-server supervisor libpq-dev gcc && \
-    ln -s /usr/bin/nodejs /usr/bin/node
+FROM praekeltfoundation/django-bootstrap:latest
```

Comment:
I don't think that we want to be building off of `latest`, in order to prevent this build breaking too much from upstream changes.

There's a `py3.6-stretch` tag that we can probably use to upgrade to stretch instead.

Reply:
👍